flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/runtime/stream/sql/AggregateITCase.scala

```diff
@@ -346,6 +346,73 @@ class AggregateITCase(aggMode: AggMode, miniBatch: MiniBatchMode, backend: State
     assertThat(sink.getRetractResults.sorted).isEqualTo(expected.sorted)
   }
 
+  @TestTemplate
+  def testGroupRowsByColumnOrdinals1(): Unit = {
```

Comment:
Can we have more meaningful names for tests?

For example `testGroupRowsBySingleColumnOrdinals`